packages/events/src/service/events/events.ts

```diff
@@ -86,11 +87,12 @@ export async function deleteEvent(
     })
   }
 
-  await deleteEventAttachments(token, event)
-
   const { id } = event
-  await collection.deleteOne({ id })
+  await deleteEventAttachments(token, event)
```

Comment:
I switched up the order here just so attachments are deleted before the actual record is. In the case of something failing after the record is deleted from MongoDB, the handler would otherwise start returning 404